app/controllers/rents_controller.rb

```diff
@@ -41,29 +40,23 @@ def update
 
   def destroy
     @rent.destroy
-    redirect_to :bil, notice: alert_destroy(@rent)
+    redirect_to :rents, notice: alert_destroy(Rent)
   end
 
   # Index page available to logged in users.
   def index
-    if current_user.present?
-      @rents = current_user.rents.order('d_from desc')
-      render action: 'index'
-    else
-      redirect_to(action: 'main')
-    end
+    @rents = current_user.rents.order("d_from desc")
   end
 
-
   private
 
-  # @rents: used to show rents under the form when creating new.
-  def set_rents
-    id = params[:id] || nil
-    @rents = Rent.active.date_overlap(Time.zone.now, Time.zone.now+30.days, id).limit(10).ascending
+  def rent_params
+    params.require(:rent).permit(:d_from, :d_til, :purpose,
+                                 :disclaimer, :council_id)
   end
 
-  def rent_params
-    params.require(:rent).permit(:d_from, :d_til, :purpose, :disclaimer, :council_id)
+  def load_terms
+    constant = Constant.find_by(name: "rents_disclaimer_id")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.